src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -235,7 +266,12 @@ private void createEntity(
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(eventEntity);
 
-    json.put("success", "true");
+    if (isEventEditing.equals("yes")) {  
+      json.put("success", "edit");
```

Comment:
Mixing boolean values with strings like "edit" is a code smell that an enum type might be more appropriate.